src/styles/_generalSettings.scss

```diff
@@ -38,13 +38,21 @@ body {
 }
 
 .heart__icon {
-  position: absolute;
-  height: 50px;
-  width: 50px;
   background: url("../assets/icons/Imgheart.svg") no-repeat center;
+}
+
+.join-to-event__icon {
+  background: url("../assets/icons/ImgJoinToEvent.svg") no-repeat center;
+}
+
+.heart__icon,
```

Comment:
Why these styles places into generalSettings?

Reply:
I moved it to styles icons